lib/app.tsx

```diff
@@ -51,7 +53,11 @@ export default class App extends PureComponent<AppProps, AppState> {
   }
 
   componentDidMount() {
-    if (this.props.autoplay) {
+    let autosave = this.props.autosaver && this.props.autosaver.restore();
+
+    if (autosave && autosave !== this.state.editorContent) {
+      this.setState({ editorContent: autosave });
+    } else if (this.props.autoplay) {
```

Comment:
Note that if we're restoring autosaved content, we don't want to auto-play here, since mis-behaving user code could slow down page load.